nf_core/utils.py

```diff
@@ -74,9 +76,9 @@ def setup_requests_cachedir():
     """
     # Only import it if we need it
     import requests_cache
-    
 
-    cachedir = os.path.join(os.getenv("HOME"), os.path.join('.nfcore', 'cache'))
+    pyversion = '.'.join(str(v) for v in sys.version_info[0:3])
```

Comment:
Sorry, I could not resist to be pedantic. Expressiveness ftw 🎉 

```suggestion
    pyversion = "{major}.{minor}.{patch}".format(
                major=sys.version_info[0],
                minor=sys.version_info[1],
                patch=sys.version_info[2]
                )
```